client/src/components/query-builder/query-builder.tsx

```diff
@@ -9,7 +9,7 @@ import {
   Select,
```

Comment:
All of the updates to this file were just eslint fixes. It should be safe to ignore.